EIPS/eip-6239.md

```diff
@@ -0,0 +1,167 @@
+---
+eip: 6239
+title: Semantic Soulbound Tokens
+description: Adding RDF triples to EIP-721 and EIP-5192 metadata to capture the social meaning of the Token
+author: Jessica Chang (@JessicaChg)
+discussions-to: https://ethereum-magicians.org/t/eip-6239-semantic-soulbound-tokens/12334
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-12-30
+requires: 165, 721, 5192
+---
+
+## Abstract
+
+This proposal extends [EIP-721](./eip-721.md) and [EIP-5192](./eip-5192.md) by introducing a standard for adding RDF triples to Soulbound Tokens (‘SBTs‘) metadata.
+
+Soulbound Token represents the commitments, credentials, and affiliations of accounts. Resource Description Framework (‘RDF’) is a standard data model developed by the World Wide Web Consortium (‘W3C’) and is used to represent information in a structured, machine-readable format in triples consisting of a subject, a predicate, and an object (the ‘RDF triple’). These triples can be combined and linked together to represent more complex information and relationships. Semantic SBTs are built on existing [EIP-721](./eip-721.md) and [EIP-5192](./eip-5192.md) standards to include RDF triples in metadata to capture and store the meaning of social metadata as a network of accounts and attributes.
```

Comment:
Cutting a bit more to keep the abstract short:

```suggestion
A Soulbound Token represents the commitments, credentials, and affiliations of accounts. RDF is a standard data model developed by the World Wide Web Consortium (‘W3C’) and is used to represent information in a structured format. Semantic SBTs are built on existing [EIP-721](./eip-721.md) and [EIP-5192](./eip-5192.md) standards to include RDF triples in metadata to capture and store the meaning of social metadata as a network of accounts and attributes.
```

Reply:
Done.

Reply:
Changed as suggested.